test/run.el

```diff
@@ -108,6 +108,11 @@ Node `(ert)Test Selectors' for information about test selectors."
   (let* ((load-prefer-newer t)
          (source-directory (locate-dominating-file flycheck-runner-file "Cask"))
          (pkg-rel-dir (format ".cask/%s/elpa" emacs-version)))
+
+    ;; standardise on the C locale so we can be sure that neither of
+    ;; gcc and gfortran will output smartquotes
+    (setenv "LC_ALL" "C")
```

Comment:
On Sat, Feb 27, 2016 at 12:06:58PM -0800, Sebastian Wiesner wrote:
> Is that enough? Shouldn't we set $LANG and $LC_MESSAGES as well? Sorry if
> that's a stupid question, but I'm not at all familiar with the semantics of all
> these locale variables.

My understanding of the locale variables is that LC_ALL is designated as
an override variable, so we only need to set that one.  The gcc manpage
suggests that LC_ALL is enough (and indeed, the test suite passes).

On Sat, Feb 27, 2016 at 12:21:01PM -0800, Sebastian Wiesner wrote:
> Please squash your commits and start the commit message with a capitalised
> letter (see our commit message guidelines).

Done.

-- 
Sean Whitton